app/views/about/index.html.erb

```diff
@@ -60,7 +60,8 @@
             <% end %>
             </tbody>
           </table>
-          <%= _('To enable a provider, either install the OS package (e.g. foreman-libvirt) or enable the bundler group for development setup (e.g. libvirt).') %>
+          <%= _('You can enable additional compute resources. For more information, see') %>
```

Comment:
Splitting sentences in translation is always a bad a idea and must be avoided. Please use string interpolation instead. See https://projects.theforeman.org/projects/foreman/wiki/Translating. Though I'm not 100% sure what the current best practice around links is.